### PR TITLE
chore: documentation around new release process + release cut time @W-10785024@

### PR DESCRIPTION
**Pull request by jag-sfdc**
### What does this PR do?
documentation around new release process + release cut time

### What issues does this PR fix or reference?
@W-10785024@

### Functionality Before
N/A

### Functionality After
No change in functionality

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -623,7 +623,7 @@ workflows:
   cut-release:
     triggers:
       - schedule:
-          cron: '0 4 * * 3'
+          cron: '0 13 * * 1'
```
Comment (jag-sfdc) [inline .circleci/config.yml:626]: Changed cron to cut release from Tuesday evening to Monday morning.

Comment (gbockus-sf) [reply .circleci/config.yml:626]: reaching waaay back for my cron time knowledge..so from Wed 5am to monday 2pm.  What time zone do these run in ?

**Review by gbockus-sf — APPROVED** [review]
LGTM

File: .circleci/config.yml (hunk 1 above)
Comment (jag-sfdc) [reply .circleci/config.yml:626]: They run in GMT. I simply use https://crontab.guru/

**Pull request merged**